File: node.py
class node:
    collector = 0.0
    connections = []

    def __init__(self,connections):
        self.connections = connections
        self.collector = 0.0
    def set_collector(self, collect):
        self.collector = collect
    def __str__(self):
        return str(self.collector)

File: node.cpp
#include "node.h"
#include <vector>
#include <iostream>
#include <string>

using namespace std;



	
	node::node() {
	};//basic constructor for the node class

	node::node(vector<node*> layer) {
		  this->connections = layer;
	}
	void node::set_collector(double value) {
		this->collector = value;
	}
	void node::set_connections(vector <node*> connections) {
		this->connections = connections;
	}
	

